README.md

```diff
@@ -1 +1,82 @@
-# Leaderboard-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard Project
+
+> In this project I have set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I have developed a first working version of the app following a wireframe. I have consumed the Leaderboard API using JavaScript async and await and added some styling.
+
+
+###### Languages 
+- HTML
+- CSS
+- JAVASCRIPT
+###### Tools  
+- Github
+###### Code Editor
+- VS code
+###### Packages 
+- NPM
+- Node
+- Webpack
+
+## Live Demo (if available)
+
+[View Live Demo Here](https://rn486.github.io/Leaderboard-Project/)
+
+
+## Getting Started
+
+>To get a local copy up and running follow these simple example steps.
+
+-Clone this repository with
+
+<code> git@github.com:Rn486/Leaderboard-Project.git </code>
```

Comment:
- [ ] [OPTIONAL] The commend here is `git clone git@github.com:Rn486/Leaderboard-Project.git`, kindly consider correcting it so the user can clone your repo

Reply:
This is the SSH link from GitHub, it should be working correctly.